iommu-group.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 shopt -s nullglob
-for d in /sys/kernel/iommu_groups/*/devices/*; do
-    n=${d#*/iommu_groups/*}; n=${n%%/*}
-    printf 'IOMMU Group %s ' "$n"
-    lspci -nns "${d##*/}"
+for g in /sys/kernel/iommu_groups/*; do
+    echo "IOMMU Group ${g##*/}:"
+    for d in $g/devices/*; do
+        echo -e "\t$(lspci -nns ${d##*/})"
```

Comment:
```suggestion
    for d in "$g"/devices/*; do
        echo -e "\t$(lspci -nns "${d##*/}")"
```

@stefanleh , this is so much more readable!!! I love it!  Shellcheck had these suggestions to be deliberate around shell expansion.  It looks to have no impact on the structure of the output on my machine.  Thoughts?

Reply:
Looks fine to me 👍

Reply:
Oops, might have resolved to early (without committing). Can you please reopen?

Reply:
Now its fine. Tnx.